### PR TITLE
Prevent twig extension XSS vulnerabilities

### PR DESCRIPTION
**Pull request by MKodde**
During the introduction of twig in engineblock. Some XSS vulnerabilities have been overlooked. This PR aims to fix all of these issues.

### CODE REVIEW

File: src/OpenConext/EngineBlockBundle/Twig/Extensions/Extension/Feedback.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ public function __construct(EngineBlock_ApplicationSingleton $application)
     public function getFunctions()
     {
         return [
-            new TwigFunction('feedbackInfo', [$this, 'getFeedbackInfo'], ['is_safe' => ['html']]),
+            new TwigFunction('feedbackInfo', [$this, 'getFeedbackInfo']),
             new TwigFunction('flushLog', [$this, 'flushLog'], ['is_safe' => ['html']]),
```
Comment (jorissteyn) [inline src/OpenConext/EngineBlockBundle/Twig/Extensions/Extension/Feedback.php]: If this function does not output anything, why mark it as "safe"?

File: theme/material/templates/layouts/scripts/language-switcher.html.twig [hunk 2]
```diff
@@ -16,10 +16,10 @@
     {% else %}
         <ul class="comp-language">
             <li>
-                <a{% if currentLanguage == 'en' %} class="selected"{% endif %} href="{{ queryStringFor('en') }}">EN</a>
+                <a{% if currentLanguage == 'en' %} class="selected"{% endif %} href="{{ queryStringFor('en')|escape('url') }}">EN</a>
```
Comment (jorissteyn) [inline theme/material/templates/layouts/scripts/language-switcher.html.twig]: This is not correct, see https://twig.symfony.com/doc/2.x/filters/escape.html:

    url: escapes a string for the URI or parameter contexts. This should not be used to escape an entire URI; only a subcomponent being inserted.

In HTML, hrefs shouldnt' be url encoded but must be escaped for HTML (attr) output. So escape('html_attr') is the only valid way of escaping here. Applies to other usages of `|escape('url')` as well, of course.

Comment (jorissteyn) [inline theme/material/templates/layouts/scripts/language-switcher.html.twig]: Only the query parameters should be URL-encoded, and the twig function takes care of that. The attribute value should be escaped just like any other attribute value: using `html_attr`.

File: src/OpenConext/EngineBlockBundle/Twig/Extensions/Extension/Feedback.php (hunk 1 above)
Comment (MKodde) [reply src/OpenConext/EngineBlockBundle/Twig/Extensions/Extension/Feedback.php]: My rationale was: it's safe if it doesn't output anything. We're talking about the `flushLog` method here right?

**Pull request merged**